apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -124,34 +143,79 @@ Response cachedHttpResponse(String cacheKey) throws IOException {
     private Builder() {
     }
 
+    /**
+     * Set the {@link OkHttpClient} to use for making network requests.
+     *
+     * @param okHttpClient the client to use.
+     * @return The {@link Builder} object to be used for chaining method calls
+     */
     public Builder okHttpClient(@Nonnull OkHttpClient okHttpClient) {
       this.okHttpClient = checkNotNull(okHttpClient, "okHttpClient is null");
       return this;
     }
 
+    /**
+     * <p>Set the API server's base url.</p>
+     *
+     * <b>Base URLs should not have a trailing slash</b>
```

Comment:
So I checked a couple of public graphQL apis and all of them seem to be not having a trailing slash. So I assume that should indeed be the case. What is your take on this? Should we add it to the docs?